Multy/Cells/AddressCell/WalletAddressTableViewCell.swift

```diff
@@ -0,0 +1,37 @@
+//Copyright 2017 Idealnaya rabota LLC
+//Licensed under Multy.io license.
+//See LICENSE for details
+
+import UIKit
+
+class WalletAddressTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var addressLbl: UILabel!
+    @IBOutlet weak var sumLbl: UILabel!             // 0.0001 BTC / 100 USD
+    @IBOutlet weak var creationTimeLbl: UILabel!    // 1.11.2017 ∙  23:15
+    
+    var wallet: UserWalletRLM?
```

Comment:
Here, and every other X-`Cell`, why is `wallet` optional? Can `WalletAddressTableViewCell` be used without wallet?

Reply:
We are setting wallet variable from previous screen after creating current screen (at the time of initializing screen this variable is empty)